pkg/encoding/address/address.go

```diff
@@ -1,21 +1,20 @@
-package crypto
+package address
 
 import (
 	"github.com/CityOfZion/neo-go/pkg/encoding/base58"
 	"github.com/CityOfZion/neo-go/pkg/util"
 )
 
-// AddressFromUint160 returns the "NEO address" from the given
-// Uint160.
-func AddressFromUint160(u util.Uint160) string {
+// EncodeUint160 returns the "NEO address" from the given Uint160.
+func EncodeUint160(u util.Uint160) string {
 	// Dont forget to prepend the Address version 0x17 (23) A
 	b := append([]byte{0x17}, u.BytesBE()...)
 	return base58.CheckEncode(b)
 }
 
-// Uint160DecodeAddress attempts to decode the given NEO address string
+// DecodeUint160 attempts to decode the given NEO address string
 // into an Uint160.
-func Uint160DecodeAddress(s string) (u util.Uint160, err error) {
+func DecodeUint160(s string) (u util.Uint160, err error) {
```

Comment:
Same here, I'd expect `DecodeUint160` to accept `Uint160` as _an argument_ and return some form of `address` (package name).
Example from stdlib: `hex.DecodeString(string) ([]byte error)`

Reply:
`StringToUint160`? The problem here is that `address` doesn't define any type of its own, so `ToString` and `FromString` might seem a little strange.